src/ClientHandle.cpp

```diff
@@ -281,6 +351,11 @@ void cClientHandle::Authenticate(const AString & a_Name, const cUUID & a_UUID, c
 
 		ASSERT(m_Player == nullptr);
 
+		if (!BungeeAuthenticate())
+		{
+			return;
```

Comment:
Don't forget to Kick.

Reply:
BungeeAuthenticate already does that. Should it be here?

Reply:
good point, maybe we can move it out for clarity at the expense of detailed kick messages, but later